source/_patterns/basics/forms/input.hbs

```diff
@@ -3,3 +3,72 @@
   <input type="text" class="form-control" id="inlineFormInput" placeholder="Jane Doe">
   <button type="submit" class="btn btn-primary">Submit</button>
 </form>
+
+<form>
```

Comment:
I can see the form now. I agree with Sarah, it would be nice to have separate patterns for each  one.